tools/src/providers/base.py

```
@@ -0,0 +1,640 @@
+                    output_token_overrides=output_token_overrides,
+                    default_output_tokens=default_output_tokens,
+                    extra_profile_fields=extra_profile_fields,
+                    apply=apply,
+                    services_json_path=services_json_path,
+                    use_litellm=use_litellm,
+                    use_openrouter=False,  # context_window already in api_entry
+                    use_config_overrides=use_config_overrides,
+                    global_protected_profiles=global_protected_profiles,
+                    deprecation_source='OpenRouter',
+                )
+            elif use_litellm:
```

Ruff (0.15.9)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)